independent-projects/bootstrap/pom.xml

```diff
@@ -97,80 +97,36 @@
-                                            <exclude>org.jboss.logging:jboss-logmanager</exclude>
-                                            <!-- We don't want all the API's in one jar-->
-                                            <exclude>javax:javaee-api</exclude>
-                                            <!-- Prevent incompatible config from coming in -->
-                                            <exclude>org.wildfly.client:wildfly-client-config</exclude>
-                                            <exclude>org.jboss.marshalling:jboss-marshalling-osgi</exclude>
-                                            <exclude>org.jboss.spec.javax.servlet:jboss-servlet-api_3.1_spec</exclude>
-                                            <!-- The API is packaged by the implementation-->
-                                            <exclude>jakarta.json:jakarta.json-api</exclude>
-                                            <!-- Ensure none of the deps use netty-all. This forces deps to use more fine grained netty artifacts -->
-                                            <exclude>io.netty:netty-all</exclude>
-                                            <!-- Ban Log4J (use org.jboss.logmanager:log4j-jboss-logmanager instead) -->
-                                            <exclude>log4j:log4j</exclude>
-                                            <exclude>org.apache.logging.log4j:log4j-api</exclude>
-                                            <exclude>org.apache.logging.log4j:log4j-core</exclude>
-                                            <exclude>org.apache.logging.log4j:log4j-slf4j-impl</exclude>
-                                            <!-- Ban commons-logging (use org.jboss.logging:commons-logging-jboss-logging instead) -->
-                                            <exclude>commons-logging:commons-logging</exclude>
-                                            <exclude>commons-logging:commons-logging-api</exclude>
-                                            <exclude>org.springframework:spring-jcl</exclude>
-                                            <exclude>org.slf4j:jcl-over-slf4j</exclude>
-                                            <!-- Ban SLF4j implementations (use org.jboss.slf4j:slf4j-jboss-logmanager instead) -->
-                                            <exclude>org.jboss.slf4j:slf4j-jboss-logging</exclude>
-                                            <exclude>org.slf4j:slf4j-simple</exclude>
-                                            <exclude>org.slf4j:slf4j-nop</exclude>
-                                            <exclude>org.slf4j:slf4j-jdk14</exclude>
-                                            <exclude>org.slf4j:slf4j-log4j12</exclude>
-                                            <exclude>org.slf4j:slf4j-log4j13</exclude>
                                             <!-- findbugs is not required at runtime -->
                                             <exclude>com.google.code.findbugs:jsr305</exclude>
```

Comment:
These ones are additional exclusions we have here and in another independent project. I decided to keep them as is as I wasn't sure if they could cause issues elsewhere.
I tested it and both rules are applied.